lib/webauthn/attestation_statement/android_safetynet.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+require "jwt"
+require "openssl"
+require "webauthn/attestation_statement/base"
+
+module WebAuthn
+  module AttestationStatement
+    # Implements https://www.w3.org/TR/2018/CR-webauthn-20180807/#android-safetynet-attestation
+    class AndroidSafetynet < Base
+      def self.default_trust_store
+        OpenSSL::X509::Store.new.tap { |trust_store| trust_store.set_default_paths }
```

Comment:
@bdewater Did you envision this becoming limited to Google's root CA certs only at some point in the future?

Reply:
Yes we should. I dug a little into Google's docs and examples but these are disappointingly light in this regard:
- No mention on https://developer.android.com/training/safetynet/attestation.html or related pages that I could find
- The [Java code example](https://github.com/googlesamples/android-play-safetynet/blob/master/server/java/src/main/java/OfflineVerify.java) uses the [system default](https://github.com/googleapis/google-http-java-client/blob/cd5a565ab67e660b0855b5fa6d62e979686c2376/google-http-client/src/main/java/com/google/api/client/json/webtoken/JsonWebSignature.java#L433-L473) trust store.
- My C# is a bit rusty but the lack of explicit root certificate(s) being configured in [the example](https://github.com/googlesamples/android-play-safetynet/blob/d7513a54e2f28c0dcd7f8d8d0fa03adb5d87b91a/server/csharp/OfflineVerify.cs) makes me think it's in the same boat as Java.

The issue was raised in https://github.com/w3c/webauthn/issues/968 but in the end not solved in the standard nor confirmed by a Google employee. However since the DigiNotar hack Google has been moving everything under their own CA so this looks like a safe change to make regardless.